docs/environment.yml

```diff
@@ -0,0 +1,85 @@
+name: venv_rdt
+channels:
+  - defaults
+dependencies:
+  - alabaster=0.7.12=py_0
+  - babel=2.9.0=pyhd3eb1b0_0
+  - blas=1.0=openblas
+  - brotlipy=0.7.0=py39h9ed2024_1003
+  - ca-certificates=2020.12.8=hecd8cb5_0
+  - certifi=2020.12.5=py39hecd8cb5_0
+  - cffi=1.14.4=py39h2125817_0
+  - chardet=4.0.0=py39hecd8cb5_1003
+  - colorama=0.4.4=pyhd3eb1b0_0
+  - cryptography=3.3.1=py39hbcfaee0_0
+  - cycler=0.10.0=py39hecd8cb5_0
+  - docutils=0.16=py39hecd8cb5_1
+  - freetype=2.10.4=ha233b18_0
+  - idna=2.10=py_0
+  - imagesize=1.2.0=py_0
+  - jinja2=2.11.2=py_0
+  - jpeg=9b=he5867d9_2
+  - kiwisolver=1.3.0=py39h23ab428_0
+  - lcms2=2.11=h92f6f08_0
+  - libcxx=10.0.0=1
+  - libedit=3.1.20191231=h1de35cc_1
+  - libffi=3.3=hb1e8313_2
+  - libgfortran=3.0.1=h93005f0_2
+  - libopenblas=0.3.10=h0794777_0
+  - libpng=1.6.37=ha441bb4_0
+  - libtiff=4.1.0=hcb84e12_1
+  - lz4-c=1.9.2=h79c402e_3
+  - markupsafe=1.1.1=py39h9ed2024_0
+  - matplotlib=3.3.2=hecd8cb5_0
+  - matplotlib-base=3.3.2=py39h13cdb87_0
+  - ncurses=6.2=h0a44026_1
+  - numpy=1.19.2=py39h0fa1045_0
+  - numpy-base=1.19.2=py39h3a452eb_0
+  - olefile=0.46=py_0
+  - openssl=1.1.1i=h9ed2024_0
+  - packaging=20.8=pyhd3eb1b0_0
+  - pandas=1.2.0=py39hb2f4e1b_0
+  - pillow=8.1.0=py39h5270095_0
+  - pip=20.3.3=py39hecd8cb5_0
+  - pycparser=2.20=py_2
+  - pygments=2.7.3=pyhd3eb1b0_0
+  - pyopenssl=20.0.1=pyhd3eb1b0_1
+  - pyparsing=2.4.7=py_0
+  - pysocks=1.7.1=py39hecd8cb5_0
+  - python=3.9.1=h88f2d9e_2
+  - python-dateutil=2.8.1=py_0
+  - pytz=2020.5=pyhd3eb1b0_0
+  - readline=8.0=h1de35cc_0
+  - requests=2.25.1=pyhd3eb1b0_0
+  - scipy=1.5.2=py39h4420a3a_0
+  - seaborn=0.11.1=pyhd3eb1b0_0
+  - setuptools=51.1.2=py39hecd8cb5_3
+  - six=1.15.0=py39hecd8cb5_0
+  - snowballstemmer=2.0.0=py_0
+  - sphinx=3.4.3=pyhd3eb1b0_0
+  - sphinx_rtd_theme=0.4.3=py_0
+  - sphinxcontrib-applehelp=1.0.2=py_0
+  - sphinxcontrib-devhelp=1.0.2=py_0
+  - sphinxcontrib-htmlhelp=1.0.3=py_0
+  - sphinxcontrib-jsmath=1.0.1=py_0
+  - sphinxcontrib-qthelp=1.0.3=py_0
+  - sphinxcontrib-serializinghtml=1.1.4=py_0
+  - sqlite=3.33.0=hffcf06c_0
+  - tk=8.6.10=hb0a8c7a_0
+  - tornado=6.1=py39h9ed2024_0
+  - tzdata=2020d=h14c3975_0
+  - urllib3=1.26.2=pyhd3eb1b0_0
+  - wheel=0.36.2=pyhd3eb1b0_0
+  - xz=5.2.5=h1de35cc_0
+  - zlib=1.2.11=h1de35cc_3
+  - zstd=1.4.5=h41d2c2f_0
+  - pip:
+    - cvxpy==1.1.7
```

Comment:
why are libs like `cvxpy`, `sklearn`, etc... required for running RDT?
Or do you need all project dependencies to run RDT?

Reply:
You need all